examples/jsm/interactive/HTMLMesh.js

```diff
@@ -268,25 +283,122 @@ function html2canvas( element ) {
 
 			const backgroundColor = style.backgroundColor;
 
+			// Get the border of the element used for fill and border
+			roundRectPath(x, y, width, height, parseFloat(style.borderRadius) );
```

Comment:
So the idea is that `roundRectPath()` renders both round and non-round, right?

If so, I would just call it `buildRectPath()` instead.

Reply:
Okay :)

Reply:
Done.